docs/DevGuide.md

```diff
@@ -80,3 +80,16 @@ EXEC : gyp verb `which` failed error : not found: python2 [E:\gh\chcosta\arcade-
 EXEC : gyp verb `which` failed  python2 error : not found: python2 [E:\gh\chcosta\arcade-services\src\Maestro\maestro-angular\maestro-angular.proj]
 ```
   Make sure python 2.7 is installed and on your path.  You may have to copy `python27\python.exe` to `python27\python2.exe`
+- if DNS Service is enabled for the cluster, Service Fabric will automaticaly edit the DNS settings for your machine and add SF as first DNS server which sometimes causes issues with DNS resolution and you may not be able to access any webpages. There is information about the issue in https://github.com/microsoft/service-fabric/issues/124
```

Comment:
Can you indent the lines above (77-82) to make it more obvious they belong to that bullet there?

Reply:
like this?